contrib/macdeploy/detached-sig-create.sh

```diff
@@ -23,7 +23,7 @@ fi
 rm -rf ${TEMPDIR} ${TEMPLIST}
 mkdir -p ${TEMPDIR}
 
-${CODESIGN} -f --file-list ${TEMPLIST} "$@" "${BUNDLE}"
+${CODESIGN} --options runtime -f --file-list ${TEMPLIST} "$@" "${BUNDLE}"
```

Comment:
Is there any reason why we can't pass `--strict` to `codesign` here as well?
```bash
     --strict options
             When validating code, apply additional restrictions beyond the defaults.

             symlinks  Check that symbolic links inside the code bundle point to sealed files inside its bundle.
                       This means that broken symbolic links are rejected, as are links to places outside the bundle
                       and to places that are not, for whatever reason, sealed by the signature.

             sideband  Check that no resource forks, Finder attributes, or similar sideband data is present in the
                       signed code.  This is now automatically enforced by signing operations.
             Options can be specified as a comma-separated list. Use plain --strict or --strict=all to be as strict
             as possible. Note that --strict=all may include more checking types over time.
             Not all strictness check make sense in all circumstances, which is why these behaviors are not the
             defualt.
```

Reply:
Isn't that for validation only?

Reply:
Yes that looks correct.